risc0/zkvm/README.md

```diff
@@ -27,3 +27,23 @@ channel](https://www.youtube.com/@risczero) has many more videos as well.
   see if there's a workaround, and if you're using a workaround, you can track
   when it gets resolved to a permanent solution.
 * And more on [the RISC Zero website](https://www.risczero.com/)!
+
+# Crate Feature Flags
+
+The following crate feature flags are available. They are configured in your `Cargo.toml`.
+
+* cuda
+  * Optional (default is CPU only)
+  * Turns on GPU acceleration for guest code with CUDA
```

Comment:
This isn't acceleration for guest code, it's acceleration specifically for the prover. Basically `prove` must be set when `metal` or `cuda` is selected.